app/controllers/collections_controller.rb

```diff
@@ -87,4 +117,39 @@ def set_collection
   def collection_params
     params.require(:collection).permit(:title, :description, :image, :image_url, :public, {:keywords => []}, {:material_ids => []}, {:event_ids => []})
   end
+
+  # once associations are properly set up to use polymorphic items
```

Comment:
probably this can really only be removed if we choose to have a single page for curating events and materials at the same time